docs/plans/2026-02-15-feat-prd-schema-versioning-plan.md

```
@@ -0,0 +1,125 @@
+## Quality Gates
+
+- [x] Manual test: create new PRD, verify schema_version present
+- [x] Manual test: load old PRD, verify auto-migration
+
+## Dependencies & Risks
+
+**Dependencies:**
+- `jq` for JSON manipulation (already required)
+
+**Risks:**
+| Risk | Mitigation |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)